IFC4x3/Properties/s/SupportClassification_09LzP6dcn6OAdINBkUxEqS/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also, Classification Association?

Reply:
I agree, but involving @JontyDunn in this conversation.

Reply:
Rename to SupportClass. @JontyDunn or @hessf-ilf might improve the description, e.g. with an example. Right now it is "Specifies a class defining the support measures for underground excavation."